backend/migrator/migration/prod/2.15/0012##query_history.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE query_history (
+    id BIGSERIAL PRIMARY KEY,
+    row_status row_status NOT NULL DEFAULT 'NORMAL',
+    creator_id INTEGER NOT NULL REFERENCES principal (id),
+    created_ts BIGINT NOT NULL DEFAULT extract(epoch from now()),
+    updater_id INTEGER NOT NULL REFERENCES principal (id),
```

Comment:
Forgot to mention. We can also get rid of updatee_id and ts, and update trigger. This will not change.

Reply:
LGTM, updated